mycli/main.py

```diff
@@ -489,6 +497,18 @@ def _connect():
             self.echo(str(e), err=True, fg='red')
             exit(1)
 
+
+    def get_password_from_file(self, password_file):
+        password_from_file = None
+        if password_file:
+            if (os.path.isfile(password_file) or \
+                stat.S_ISFIFO(os.stat(password_file).st_mode)) \
+                and os.access(password_file, os.R_OK):
+                with open(password_file) as fp:
+                    password_from_file = fp.read()
```

Comment:
Should we strip trailing `\n`, `\r\n` and space on both sides?

Reply:
excellent idea. Will do